cli/src/cli_builder.rs

```diff
@@ -269,13 +269,19 @@ impl<'a> NewCli<'a> {
         let mut node = &runner;
         let mut sm = &matches;
         while let Some((sub_name, sub_matches)) = sm.subcommand() {
-            node = node.children.get(sub_name).unwrap();
```

Comment:
Is this an improvement? Literally this would be the violation of a program invariant. I don't believe it's testable

Reply:
Fair, replaced with `expect`.